lib/internal/modules/cjs/loader.js

```diff
@@ -654,8 +654,8 @@ function getDefaultExtensions() {
   const tsEnabled = getOptionValue('--experimental-strip-types');
   if (tsEnabled) {
     extensions = ArrayPrototypeFilter(extensions, (ext) =>
-      ext !== '.ts' || Module._extensions['.ts'] !== loadTS ||
-      ext !== '.cts' || Module._extensions['.ts'] !== loadCTS,
+      (ext !== '.ts' || Module._extensions['.ts'] !== loadTS) &&
```

Comment:
Can you add a comment to this function and explain what it does?

Reply:
done